src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -144,6 +161,26 @@ public PlatformDetectorResult Detect(RepositoryContext context)
             var packageDir = GetPackageDirectory(context);
             var virtualEnvName = GetVirtualEnvironmentName(context);
             var isPythonPackageCommandEnabled = _commonOptions.ShouldPackage;
+            var pythonPackageWheelType = GetPythonPackageWheelType(context);
+
+            if (!isPythonPackageCommandEnabled && !string.IsNullOrWhiteSpace(pythonPackageWheelType))
+            {
+                throw new InvalidUsageException($"Option '{PythonPackageWheelPropertyKey}' cann't exist" +
+                    $"without package command being enabled. Please provide --package along with wheel type");
+            }
+
+            if (isPythonPackageCommandEnabled &&
+                !string.IsNullOrWhiteSpace(pythonPackageWheelType) &&
+                !string.Equals(pythonPackageWheelType.ToLower(), "universal"))
```

Comment:
If we went with the array model, this is where we would validate that the given type was in our list.

Reply:
im not sure what is arraymodel you are referring here

Reply:
as discussed, at this moment  we would go with non-array model, when the time comes to support new value we can add the array model. Similar thing what we do for zipoption and taroption ... we could have simply used one option and have an array with 2 elements zip and tar ... so im keeping that as it is for now